dftimewolf/cli/dftimewolf_recipes.py

```diff
@@ -183,6 +183,11 @@ def ParseArguments(self, arguments):
     print('Loaded recipe {0:s} with {1:d} modules'.format(
         self._recipe['name'], number_of_modules))
 
+  def RunPreflights(self):
+    """Runs preflight modules."""
+    print('Running preflights...')
+    self._state.RunPreflights()
```

Comment:
Consider inlining this (and other functions that follow this pattern) in a future PR.

Reply:
Ack, added an issue for this.